api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillEsRepo.kt

```diff
@@ -62,6 +71,32 @@ class CustomRichSkillQueriesImpl @Autowired constructor(override val elasticSear
     CustomRichSkillQueries {
     override val javaClass = RichSkillDoc::class.java
 
+    override fun getUuidsFromApiSearch(
```

Comment:
Are we adding unit tests for this new logic?

Reply:
Yeah, there are tests for two cases, when api search contains uuids or query.

Reply:
I see them, I recommend to move those into the RichSkillEsRepoTest class instead of in CollectionEsRepoTest